src/main/webapp/find-events.html

```diff
@@ -39,5 +39,49 @@ <h1>find events</h1>
         </div>
       </div>
     </div>
+
+    <div id="modal-background">
+      <div id="modal-create-post">
+        <button id="modal-close" type="button">&#10005;</button>
+        <h1 id="modal-title">Add a Post</h1>
+        <form id="modal-form">
+          <input type="text" id="modal-org-name" name="organizationName" placeholder="organization name" required><br>
+
+          <span class="modal-span">
+            <p id="modal-time-text">date:</p>
+            <input type="number" id="modal-month" name="month" placeholder="mm" min="1" max="12" step="1">
+            <p id="modal-slash">/</p>
+            <input type="number" id="modal-day" name="day" placeholder="dd" min="1" max="31" step="1">
+            <p id="modal-slash">/</p>
+            <input type="number" id="modal-year" name="year" placeholder="yy" min="20" max="30" step="1">
+          </span>
+
+          <span class="modal-span">
+            <p id="modal-time-text">start:</p>
+            <input type="number" id="modal-start-hour" name="startHour" placeholder="h" min="1" max="12" step="1">
+            <input type="number" id="modal-start-minute" name="startMinute" placeholder="m" min="0" max="60" step="1">
+            <select id="start-am-or-pm" name="AMorPM">
+              <option value="am" selected>am</option>
+              <option value="pm">pm</option>
+            </select>
+          </span>
+          <span class="modal-span">
```

Comment:
I think there might be a newline missing here

Reply:
Ah thank you